### PR TITLE
1260 dataflow domain smedilol 1

### PR DESCRIPTION
**Pull request by smedilol**
Small changes for 1260_DataflowDomain

### CODE REVIEW

File: TypeCobol/Compiler/Scopes/Scope.cs [hunk 1]
```diff
@@ -77,16 +77,16 @@ public IEnumerator<T> GetEnumerator()
...
-        /// <summary>
-        /// One Symbol constructor.
-        /// </summary>
-        /// <param name="symbol"></param>
-        public Entry(T symbol)
+			/// <summary>
+			/// One Symbol constructor.
+			/// </summary>
+			/// <param name="symbol"></param>
+            public Entry(T symbol)
```
Comment (fm-117) [inline TypeCobol/Compiler/Scopes/Scope.cs]: Inconsistent indent : mix of tabs and spaces

File: TypeCobol/Compiler/Symbols/AbstractScope.cs [hunk 2]
```diff
@@ -9,6 +9,9 @@
 
 namespace TypeCobol.Compiler.Symbols
 {
+    /// <summary>
+    /// Don't confuse with Scope class
```
Comment (fm-117) [inline TypeCobol/Compiler/Symbols/AbstractScope.cs]: Maybe a little bit more of an explanation 😄 ? "Represents any symbol that contain other symbols (i.e. ProgramSymbol or NamespaceSymbol)"

File: TypeCobol/Compiler/Symbols/VariableTypeSymbol.cs [hunk 3]
```diff
@@ -52,8 +52,8 @@ internal string[] TypePaths
         }
 
         /// <summary>
-        /// The Type of a variable whose type comes from a TYPEDEF can be set letter when
-        /// The TYPEDEF symbol is resolved. 
+        /// The Type of a variable whose type comes from a TYPEDEF can be set later when
+        /// rhe TYPEDEF symbol is resolved.
```
Comment (fm-117) [inline TypeCobol/Compiler/Symbols/VariableTypeSymbol.cs]: typo in the corrected typo "the"

File: TypeCobol/Compiler/Symbols/AbstractScope.cs (hunk 2 above)
Comment (smedilol) [reply TypeCobol/Compiler/Symbols/AbstractScope.cs]: See last changes

File: TypeCobol/Compiler/Scopes/Scope.cs (hunk 1 above)
Comment (smedilol) [reply TypeCobol/Compiler/Scopes/Scope.cs]: See last changes

**Comment by smedilol** [comment]
@mayanje Would you like that I rebase this branch before merging ?

**Pull request merged**